src/java.base/share/classes/jdk/internal/javac/PreviewFeature.java

```diff
@@ -76,7 +76,6 @@ public enum Feature {
         STRUCTURED_CONCURRENCY,
         @JEP(number=466, title="ClassFile API", status="Second Preview")
         CLASSFILE_API,
-        @JEP(number=473, title="Stream Gatherers", status="Second Preview")
         STREAM_GATHERERS,
```

Comment:
Does bootcycle fail if this enum constant is removed?

Reply:
Yep :)